app/web/src/components/AssetContributeModal.vue

```diff
@@ -0,0 +1,112 @@
+<template>
+  <Modal ref="modalRef" title="Contribute Assets">
+    <Stack>
+      <ul v-if="enableContributeButton" class="flex flex-col gap-2xs max-h-72">
+        <li
+          v-for="module in props.contributeRequest.modules"
+          :key="module.schemaId"
+          class="flex px-1 items-center"
+        >
+          <span class="pr-2 select-none">•</span>
+          {{ module.name }}
+          <span class="text-2xs italic text-neutral-500 ml-xs"> FOO </span>
+        </li>
```

Comment:
At the time of writing, I am unable to see this yet because I cannot get a new asset to show `canContribute`. Maybe I was missing something in my local setup though.